config/templates/cloud-config-etcd

```diff
@@ -86,6 +87,17 @@ coreos:
 
 
 write_files:
+
+  - path: /etc/environment
+    permissions: 0644
+    content: |
+      COREOS_PUBLIC_IPV4=$public_ipv4
+      COREOS_PRIVATE_IPV4=$private_ipv4
+      ETCDCTL_CA_FILE=/etc/etcd2/ssl/ca.pem
+      ETCDCTL_CERT_FILE=/etc/etcd2/ssl/etcd-client.pem
+      ETCDCTL_KEY_FILE=/etc/etcd2/ssl/etcd-client-key.pem
+      ETCDCTL_ENDPOINT=
```

Comment:
I'm just curious but may I ask why you've explicitly emptied `ETCD_ENDPOINT` here?
Just to note that we have to explicitly set it when running `etcdctl`?